roles/tftp-server/tasks/main.yml

```diff
@@ -11,10 +18,27 @@
     remote_src: true
     creates: /srv/tftp/pxelinux.0
 
+- name: create dir for configs
+  file:
+    path: "{{ web_root }}/{{ item }}"
+    state: directory
+    recurse: true
+    # owner:  "{{ user_name }}"
+    # group:  "{{ user_name }}"
+  with_items:
+  - lc
+  - d-i/xenial
+
 - name: write preseed.cfg
   template:
     src: preseed.cfg.j2
-    dest: /srv/tftp/preseed.cfg
+    # dest: /srv/tftp/d-i/xenial/preseed.cfg
+    dest: "{{ web_root }}/d-i/xenial/preseed.cfg"
+
+- name: push late_command.sh
+  copy:
+    src: files/late_command.sh
+    dest: "{{ web_root }}/lc/late_command.sh"
```

Comment:
lc for late command.  
2 letters because it is easy to type after wget.
but I don't find myself typing it as often as I used to.

dir to keep relevant files seperate from install kernels, menus, preseed...

ld for late dir? ;)

I could go with late, or late_command if you really care.